command.go

```diff
@@ -59,6 +59,16 @@ type Command struct {
 	Options     OptionSet
 	Annotations Annotations
 
+	// Tool is the name of the MCP tool this command provides.
+	// If set, the command can be invoked via MCP as a tool.
+	// Tool and Resource are mutually exclusive.
+	Tool string
+
+	// Resource is the URI of the MCP resource this command provides.
+	// If set, the command can be accessed via MCP as a resource.
+	// Tool and Resource are mutually exclusive.
+	Resource string
+
```

Comment:
I'd suggest maybe either prefix these with `MCP` or put them into a struct of MCP-related options.